service-wcs/src/main/java/org/oskari/wcs/geotiff/IFD.java

```diff
@@ -0,0 +1,383 @@
+package org.oskari.wcs.geotiff;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+public class IFD {
```

Comment:
Add some comment what this class is used for. Maybe a link to describe Geotiff IFD? I found this one but not sure if there's a better one https://www.geospatialworld.net/article/geotiff-a-standard-image-file-format-for-gis-applications/